README.md

```diff
@@ -0,0 +1,58 @@
+# Flatpak for DOSBox
+
+## Installation
+
+This Flatpak is available on
+[Flathub](https://flathub.org/apps/details/com.dosbox.DOSBox).
+After following the [Flatpak setup guide](https://flatpak.org/setup/),
+you can install it by entering the following command in a terminal:
+
+```bash
+flatpak install flathub com.dosbox.DOSBox -y
+```
+
+Once the Flatpak is installed, you can run DOSBox using your desktop environment's
+application launcher.
+
+## Updating
+
+This Flatpak follows the latest stable DOSBox version.
+To update it, run the following command in a terminal:
+
+```bash
+flatpak update
+```
+
+## Limitations
+
+- For security reasons, this Flatpak is sandboxed and only has access to the
+  user's Documents folder. You should place any files you need within DOSBox in
+  that folder (or in a subfolder).
+
+Please [create an issue](https://github.com/flathub/com.dosbox.DOSBox/issues/new)
+if you find any other limitations that should be documented here.
+
+## Configuration files
+
+Under the default Flatpak configuration, the DOSBox configuration files are
+located in `$HOME/.var/app/com.dosbox.DOSBox/.dosbox/`. To access it with a
```

Comment:
I would probably change the path to `~/.var/app/com.dosbox.DOSBox/.dosbox/` that also works in Nautilus. Otherwise LGTM. :-)

Reply:
Done.